<?xml version="1.0"?>
<launch>
  <arg name="world_file" default="$(find moby_stage)/worlds/intersection.world"/>
  <param name="/use_sim_time" value="true"/>

  <!--  ================ Stage ================  -->
  <node pkg="stage_ros" type="stageros" name="stageros" args="$(arg world_file)">
    <param name="base_watchdog_timeout" value="0.5"/>
    <remap from="odom" to="odom"/>
    <remap from="base_pose_ground_truth" to="base_pose_ground_truth"/>
    <remap from="/robot_0/cmd_vel" to="obs0/cmd_vel"/>
    <remap from="/robot_1/cmd_vel" to="obs1/cmd_vel"/>
    <remap from="/robot_0/odom" to="obs0/odom"/>
    <remap from="/robot_1/odom" to="obs1/odom"/>
  </node>
</launch>
